docs/docs/addons-two-way-binding-helpers.md

```diff
@@ -1,16 +1,27 @@
 ---
 id: two-way-binding-helpers
-title: Two-Way Binding Helpers
-permalink: docs-old/two-way-binding-helpers.html
-prev: animation.html
-next: test-utils.html
+title: Two-way Binding Helpers (Legacy)
+layout: docs
+category: Add-Ons
+permalink: docs/two-way-binding-helpers.html
+prev: pure-render-mixin.html
+next: update.html
 ---
 
-`ReactLink` is an easy way to express two-way binding with React.
-
 > Note:
 >
-> ReactLink is deprecated as of React v15. The recommendation is to explicitly set the value and change handler, instead of using ReactLink.
+> 'react-addons-linked-state-mixin' is a legacy add-on. The recommendation is to explicitly set the value and change handler, instead of using `LinkedStateMixin`.
+
+**Importing**
+
+```javascript
+import LinkedStateMixin from 'react-addons-linked-state-mixin' // ES6
+var LinkedStateMixin = require('react-addons-linked-state-mixin') // ES5
+```
+
+## Overview
+
+`ReactLink` is an easy way to express two-way binding with React.
```

Comment:
`LinkedStateMixin`. There is no `ReactLink` in public API, it's a completely internal name (no idea how it got here 😄 ).